docs/getting-started/index.hbs

```diff
@@ -0,0 +1,149 @@
+---
+title: Introduction
+nav: side/getting-started/introduction
+order: 1
+lunr: true
+fontUrl: https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700
+cssUrl: https://design.axa.com/toolkit/dist/all.css
+jsUrl: https://design.axa.com/toolkit/dist/all.js
+metaExplanationUrl: https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do
+toc:
+  - title: Quick Start
+    slug: quick-start
+  - title: Starter Template
+    slug: starter-template
+  - title: Important Globals
+    slug: important-globals
+---
+
+{{#> page}}
+  {{#*inline "content"}}
+
+    <h1 class="docs-h2">{{title}}</h1>
+
+    <p class="lead">
+      The Web Toolkit is the <em>CSS, JS and HTML framework</em>
+      that we use globally at <a href="https://www.axa.com">AXA</a>
+      <em>for modern and responsive web apps</em>. It is based
+      on <em>Bootstrap 3</em> and provides <em>many specific components
+      and utilities</em> for you to use. It makes developing great web apps
+      that follow our
+      <a href="{{package.homepageWebGuidelines}}">Web Guidelines</a> super easy!
+    </p>
+
+    <p>This is how to get started with the AXA Web Toolkit using
+    assets from our CDN and a simple starter template.</p>
+
+    {{> table-of-contents data=toc}}
+
+    <h2 id="quick-start" class="docs-h3">Quick Start</h2>
+
+    <p>Looking to quickly add the Web Toolkit to your project?
+    Use our CDN. Using a package manager
+    or need to download the source files?
+    <a href="{{relative '/getting-started/downloads/'}}">Check out
+    the downloads page.</a></p>
+
+    <p>Copy-paste the stylesheet <code>&lt;link&gt;</code> of the
```

Comment:
Maybe just tell them that they can download the compiled assets using this link and manage that dependency locally?
Unversioned cdn is pretty worthless anyway. Their website will change whenever we update the thing...